.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "solidity.compileUsingRemoteVersion": "v0.8.16+commit.07a7930e",
+    "solidity.packageDefaultDependenciesDirectory": "protocol/node_modules"
+}
```

Comment:
I'm not sure this should be part of the project, since vscode is not mandatory (and not even a default for our team)

Reply:
Deleted, it was included by error